<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="cs">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Control System</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fire_signal">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Fire Sigal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="power_out">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Power Out</string>
    </property>
   </widget>
   <widget class="QLabel" name="passenger1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>300</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Passenger 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="up1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="down1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>370</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Down</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="pFloor1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLCDNumber { background-color: black; }</string>
    </property>
   </widget>
   <widget class="QLabel" name="passenger2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>300</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Passenger 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="up2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>370</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="down2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>370</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Down</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="pFloor2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>330</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLCDNumber { background-color: black; }</string>
    </property>
   </widget>
   <widget class="QLabel" name="passenger3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>300</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Passenger 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="up3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>370</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="down3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>370</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Down</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="pFloor3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>330</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLCDNumber { background-color: black; }</string>
    </property>
   </widget>
   <widget class="QPushButton" name="open1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>170</y>
      <width>55</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Opne</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close1">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>170</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="obstacle1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>55</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>obstacle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fire1">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Fire</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ePanel11">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>30</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ePanel12">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>30</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ePanel13">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>30</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLabel" name="elevator1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Elevator 1</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="eFloor1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLCDNumber { background-color: black; }</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ePanel16">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>30</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ePanel14">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>30</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ePanel15">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>30</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ePanel17">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>30</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ePanel23">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>80</y>
      <width>30</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>170</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ePanel27">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>140</y>
      <width>30</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="obstacle2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>200</y>
      <width>55</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>obstacle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ePanel21">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>30</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="open2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>170</y>
      <width>55</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Opne</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ePanel22">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>30</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="elevator2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Elevator 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fire2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>200</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Fire</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ePanel24">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>110</y>
      <width>30</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ePanel25">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>110</y>
      <width>30</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="eFloor2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLCDNumber { background-color: black; }</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ePanel26">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>110</y>
      <width>30</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="open3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>170</y>
      <width>55</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Opne</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ePanel32">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>80</y>
      <width>30</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ePanel34">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>110</y>
      <width>30</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="obstacle3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>200</y>
      <width>55</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>obstacle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>170</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ePanel31">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>80</y>
      <width>30</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ePanel36">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>110</y>
      <width>30</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ePanel37">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>140</y>
      <width>30</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QLabel" name="elevator3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Elevator 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ePanel35">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>110</y>
      <width>30</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ePanel33">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>80</y>
      <width>30</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fire3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>200</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Fire</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="eFloor3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLCDNumber { background-color: black; }</string>
    </property>
   </widget>
   <widget class="QPushButton" name="help1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QPushButton" name="help2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>140</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QPushButton" name="help3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>140</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QPushButton" name="overload1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OverLoad</string>
    </property>
   </widget>
   <widget class="QPushButton" name="overload2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>230</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OverLoad</string>
    </property>
   </widget>
   <widget class="QPushButton" name="overload3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>230</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OverLoad</string>
    </property>
   </widget>
   <widget class="QPushButton" name="help">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>696</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
